From: Shiji Yang <yangshiji66@outlook.com>
Date: Tue, 6 May 2025 08:11:10 +0800
Subject: [PATCH] ARM: dts: qcom: ipq4019: add reference label for PCIe bridge
 node

Some devices have an ath10k PCIe peripheral. Add a label so that
the bridge node can be referenced.

Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
---
 arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi
+++ b/arch/arm/boot/dts/qcom/qcom-ipq4019.dtsi
@@ -470,7 +470,7 @@
 
 			status = "disabled";
 
-			pcie@0 {
+			pcie_bridge0: pcie@0 {
 				device_type = "pci";
 				reg = <0x0 0x0 0x0 0x0 0x0>;
 				bus-range = <0x01 0xff>;
